src/components/NavBar/SearchBarDropdown.tsx

```diff
@@ -60,16 +62,15 @@ export const SearchBarDropdownSection = ({
               isHovered={hoveredIndex === index + startingIndex}
               setHoveredIndex={setHoveredIndex}
               toggleOpen={toggleOpen}
+              index={index + startingIndex}
               traceEvent={() =>
-                sendAnalyticsEvent(EventName.NAVBAR_SEARCH_EXITED, {
-                  position: index,
-                  selected_type: 'collection',
-                  suggestion_count: suggestions.length,
+                sendAnalyticsEvent(EventName.NAVBAR_RESULT_SELECTED, {
+                  position: index + startingIndex,
                   selected_name: suggestion.name,
                   selected_address: suggestion.address,
```

Comment:
can this be more descriptive, ie: "selected_search_result_name". same with address